src/project.js

```diff
@@ -73,11 +73,24 @@ const addNewProject = (() => {
   // listener for loading projects on start
   window.onload = (() => {
     const projects = JSON.parse(localStorage.getItem('projects'));
+    const card = document.createElement('div')
+    card.classList.add('card')
     const div = document.createElement('div');
     div.innerHTML = `
       In total there are ${projects.length} projects in the LocalStorage.
+
     `;
+    card.innerHTML =`<div class='card-title'>
+    <p class='heading'>Project name: ${allProjects[0].name}</p> 
+     <button class='btn btn-success disabled addBtn' }>Add task</button>
+     <button class='btn btn-danger disabled deleteProjectBtn' data=$>Delete project</button>
+    </div>
+    <div class='card-body'>
+    <div class='card-item'}>${allProjects[0].tasks[0].name} | Due date: 01-01-2022 | priority: high <button class='btn btn-danger deleteBtn disabled'>Delete</button></div>
+    </div>`
+    console.log(allProjects[0].tasks[0].name);
```

Comment:
Let's remove `console.log`
No `console.log` committed to the repo. It's ugly, it kills performance. and it can make confidential data be visible in the browser tools.

![image](https://user-images.githubusercontent.com/31897434/91425902-a9fbc300-e853-11ea-9018-510c1ca778f2.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/